src/CorePolicyRamCloud.h

```diff
@@ -0,0 +1,49 @@
+/* Copyright (c) 2009-2016 Stanford University
+ *
+ * Permission to use, copy, modify, and distribute this software for any
+ * purpose with or without fee is hereby granted, provided that the above
+ * copyright notice and this permission notice appear in all copies.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR(S) DISCLAIM ALL WARRANTIES
+ * WITH REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED WARRANTIES OF
+ * MERCHANTABILITY AND FITNESS. IN NO EVENT SHALL AUTHORS BE LIABLE FOR
+ * ANY SPECIAL, DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
+ * WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS, WHETHER IN AN
+ * ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF
+ * OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
+ */
+
+#ifndef RAMCLOUD_COREPOLICYRAMCLOUD_H
+#define RAMCLOUD_COREPOLICYRAMCLOUD_H
+
+#include "Arachne/CorePolicy.h"
+
+#define BASE_THREAD_CLASS 0
```

Comment:
These #defines need to be documented. I think you could interleave these with the "static const" declarations below so you don't have to repeat the documentation. For that matter, why do you need both a #define and a "static const"? Let's discuss.